src/main/java/com/mycompany/mszczepienia/service/AuthService.java

```diff
@@ -105,4 +102,12 @@ public RefreshJwtResponseDto refreshToken(RefreshJwtRequestDto refreshJwtRequest
                 .refreshToken(refreshToken)
                 .build();
     }
+    @Transactional
+    public UpdatePasswordDto updatePassword(UpdatePasswordDto forgotPasswordDto){
```

Comment:
There's no need to return anything from this method - especially the password which is a sensitive data.